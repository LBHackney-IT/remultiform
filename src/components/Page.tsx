import React, { Component } from "react";

class Page extends Component {
  render(): JSX.Element {
    return <div>Content goes here!</div>;
  }
}

export default Page;
