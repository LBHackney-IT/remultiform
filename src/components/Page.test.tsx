import React from "react";
import renderer from "react-test-renderer";

import Page from "./Page";

it("renders correctly", () => {
  const component = renderer.create(<Page />);

  expect(component).toMatchSnapshot();
});
