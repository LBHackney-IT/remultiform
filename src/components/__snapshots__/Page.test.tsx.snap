// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  Content goes here!
</div>
`;
